applications/ecallmgr/src/ecallmgr_fs_route.erl

```diff
@@ -282,7 +282,7 @@ search_for_route(Section, Node, FetchId, CallId, Props) ->
             ToSet = [{<<"Context">>, hunt_context(Props)}] ++ remote_sdp(Props),
             maybe_wait_for_authz(Section, Node, FetchId, CallId
                                  ,wh_json:set_values(ToSet, JObj)
-                                )
+                                 , Props)
```

Comment:
I need the headers to be prefixed with `sip_h_` so what about bring my `get_custom_sip_headers` function from `fs_xml` to here and get all custom variables and passed them down instead of the whole Props?